dotnet/src/Connectors/Connectors.Memory.Weaviate/WeaviateVectorStoreRecordCollectionQueryBuilder.cs

```diff
@@ -44,8 +44,8 @@ public static string BuildSearchQuery<TVector>(
         {
           Get {
             {{collectionName}} (
-              limit: {{searchOptions.Limit}}
-              offset: {{searchOptions.Offset}}
+              limit: {{searchOptions.Top}}
+              offset: {{searchOptions.Skip}}
```

Comment:
Old names are still used, maybe it's correct because they are used by underlying DB.

Reply:
This reflects the underlying datastore naming, so this is correct.